docs/validators/emergency-upgrades.md

```diff
@@ -0,0 +1,83 @@
+---
+order: 5
+title: Emergency Upgrades
+---
+
+## **Introduction**
+
+This article goes into the general flow of events for an emergency upgrade, pre-upgrade and during the upgrade for all validators.
+
+Emergency upgrades are an unfortunate necessity for all validators when running a PoS node.
+
+When a vunerability is discovered and responsibily disclosed, the vunerability will be reported to one of the channels described in the [Security.md](../../SECURITY.md).
+
+Once the vulnerability is patched and tested, the core team will reach out to all validators to update their nodes with the latest version.
+
+### **Cosmos Hub Upgrade Information**
+
+Depending on the severity of the vulnerability, the core teams and validators can take different actions.
+
+The core teams will try reach out **48 hours** or more before a scheduled upgrade, so that validators have time to inspect the changes and also adjust their internal planning accordingly.
+
+If the change is **non** state-breaking then the patch can be applied in a un-cordinated manner by validators asap.
+
+If the change is **state-breaking**, that will mean that the upgrade has to be co-ordinated for all validators at a certain block or halt-height.
+
+#### **Co-ordinated Upgrades**
+
+A co-ordinated upgrade will require updating your node configuration to the correct halt-height. The code teams will communicate the correct height for all validators via Email and [Discord](https://discord.com/channels/669268347736686612/798937713474142229).
```

Comment:
```suggestion
A co-ordinated upgrade will require updating your node configuration to the correct halt-height. The core teams will communicate the correct height for all validators via Email and [Discord](https://discord.com/channels/669268347736686612/798937713474142229).
```

Reply:
```suggestion
A co-ordinated upgrade will require updating your node configuration to the correct halt-height. The relevant teams will communicate the correct height for all validators via Email and [Discord](https://discord.com/channels/669268347736686612/798937713474142229).
```

security question: how can validators confirm the authenticity of the messaging?

Reply:
Great question from @alatuszam :
> how can validators confirm the authenticity of the messaging

This is one of the reasons why I think we need to simplify the communication during the upgrade coordination.

An e-mail list or newsletter is much easier maintainable than Discord messaging. I wouldn't lock down how the communication goes just yet in this document.

Reply:
We are prioritising the email list that ICF maintains (derived from Discord), this is still limited and a lot of validators are still missing, so outreach in the near term needs to use additional channels.